### PR TITLE
fix(portal-next): fix app list

### PR DESCRIPTION
**Pull request by illichso**
## Issue

https://gravitee.atlassian.net/browse/APIM-5463

## Description

Fix app list 

## Additional context

Screenshot 2024-08-09 at 12 51 22

Screenshot 2024-08-09 at 15 11 50

https://github.com/user-attachments/assets/ff840758-2248-4263-ab53-06445e5a5fe6

https://github.com/user-attachments/assets/bc906520-42b7-4909-a1ed-a71c86d24d77

### CODE REVIEW

File: gravitee-apim-portal-webui-next/src/app/applications/applications.component.ts [hunk 1]
```diff
@@ -64,12 +63,9 @@ export class ApplicationsComponent {
...
         const data = resp.data
-          ? resp.data.map(application => ({
-              id: application.id,
-              description: application.description,
-              name: application.name,
-              picture: application._links?.picture,
-            }))
+          ? resp.data.map(application => {
+              return application;
+            })
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/src/app/applications/applications.component.ts]: Doesn't look like your map does anything?

File: gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html [hunk 2]
```diff
@@ -20,11 +20,11 @@
     <div class="app-card__header">
       <app-picture [picture]="application._links?.picture" [hashValue]="application.name" [size]="40" />
       <div class="app-card__header__content">
-        <div class="m3-title-medium">{{ application.name }}</div>
+        <div class="m3-title-medium" [matTooltip]="application.name">{{ application.name }}</div>
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html]: Only show the tooltip when the name is abbreviated.

File: gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html [hunk 3]
```diff
@@ -20,11 +20,11 @@
...
         <div class="m3-body-medium"><span i18n="@@owner">Owner: </span>{{ application.owner?.display_name }}</div>
       </div>
     </div>
-    <div class="app-card__description m3-body-medium">
+    <div class="app-card__description m3-body-medium" [matTooltip]="application.description ? application.description : ''">
```
Comment (jourdiw) [inline gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html]: Do not put tooltip on description at all. The user can view the description in the Application page to learn more.

**Review by jourdiw — CHANGES_REQUESTED** [review]
Despite different lengths of titles, the application picture should stay in the same position (upper left) instead of centering between title + owner lines.

File: gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html (hunk 3 above)
Comment (illichso) [reply gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html]: Done

File: gravitee-apim-portal-webui-next/src/app/applications/applications.component.ts (hunk 1 above)
Comment (illichso) [reply gravitee-apim-portal-webui-next/src/app/applications/applications.component.ts]: Done

File: gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html (hunk 2 above)
Comment (illichso) [reply gravitee-apim-portal-webui-next/src/components/application-card/application-card.component.html]: Done

**Review by jourdiw — APPROVED** [review]

**Pull request merged**